web-js/.env

```diff
@@ -0,0 +1,4 @@
+REACT_APP_API_TYPE="graphql"
+# REACT_APP_API_TYPE="local"
+# REACT_APP_API_TYPE="mql"
+REACT_APP_REALM_APP_ID="todo-sync-tmikv"
```

Comment:
App id should be stored in a 'realm.json' file per the spec.

[Spec: Template Starter Applications > "Each framework will require its own format"](https://docs.google.com/document/d/1gmbjUjeSULW4aI3OV9ytBqU0sypQ78H2cpIKIHr1xWM/edit?usp=sharing)

<img width="498" alt="Screen Shot 2021-04-23 at 3 29 01 PM" src="https://user-images.githubusercontent.com/52428905/115921014-f3cf9400-a448-11eb-9c8d-5994b3956638.png">


You can copy the one in the react-native template - https://github.com/mongodb-university/realm-template-apps/blob/react-native/react-native/realm.json